neuron/src/app/Neuron/Web/View.hs

```diff
@@ -49,16 +50,17 @@ import qualified Skylighting.Format.HTML as Skylighting
 import qualified Skylighting.Styles as Skylighting
 
 -- | Render the given route
-renderRoutePage :: PandocBuilder t m => Text -> Config -> Manifest -> Route a -> (ZettelGraph, a) -> NeuronWebT t m ()
-renderRoutePage neuronVersion config manifest r val =
+renderRoutePage :: PandocBuilder t m => Text -> Config -> HeadHtml -> Manifest -> Route a -> (ZettelGraph, a) -> NeuronWebT t m ()
+renderRoutePage neuronVersion config headHtml manifest r val = do
+  el "!DOCTYPE html" $ pure ()
```

Comment:
?

Reply:
btw, you can use `blank` in place of `pure ()`.

Reply:
Oh, whoops, I had forgotten about this DOCTYPE addition. Handling DOCTYPE could be its own PR, in which case I'll remove this, but I also suspect that `<!DOCTYPE html>` is the right thing for most people.

The reasoning for adding it was for loading KaTeX, which doesn't work in quirks mode. One case for including the change in this PR is that #212 is one of this PR's motivations.

Reply:
Okay, feel free to leave it in then - but do add that note (re: KaTeX) as a code comment above that.